### PR TITLE
Test Julia package against main

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

File: .github/workflows/enzyme-julia.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          - x64
+    timeout-minutes: 30 
+    steps:
+    - uses: actions/checkout@v1
+      with:
+          fetch-depth: 1
+          path: main
+    - uses: actions/checkout@v1
+      with:
+        repository: 'wsmoses/Enzyme.jl'
```
Comment (vchuravy) [inline .github/workflows/enzyme-julia.yml:30]:
```suggestion
        repository: 'wsmoses/Enzyme.jl'
        ref: 'master'
```

**Comment by vchuravy** [comment]
@tgymnich any reason for closing this? I think having downstream CI would be valuable.

**Comment by tgymnich** [comment]
Sorry I thought this was done after fixing Enzyme.ja

**Comment by tgymnich** [comment]
For future reference: #431

**Review by wsmoses — APPROVED** [review]

**Pull request merged**